CIPs/0008.md

```diff
@@ -28,32 +28,35 @@ We propose to add another claim, called the `STORAGE` claim:
 {
   "type": "STORAGE",
   "timestamp": 1588886114907,
-  "address": "https://example.com/celo_storage"
+  "address": "https://example.com/celo_storage",
+  "filterDataAddresses": ".*"
 }
 ```
 
-Alternatively, this could be set directly on the `Accounts` smart contract directly, causing off-chain storage to effectively replace the existing metadata functionality. This would have the benefit of not duplicating similar functionality.
+This http address should be referred to as the root. The expectation here is that data can accessed via `root/${dataAddress}`. A root can specify a REGEX with the `filterDataAddresses` key to indicate what kind of data it stores. An account can make multiple `STORAGE` claims, both to increase availability, as well as effectively sharding this off-chain data. This CIP does not specify what to do in case of conflicts.
```

Comment:
I tried to avoid overly complicating the specification of what kind of data a root would host. When in doubt, one could always the wild card for everything? I imagine this should cover most use cases to start?

As mentioned, I didn't feel that the CIP should specify what to do when two roots host the different data at the same path. This feels quite application specific to me